pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -173,15 +245,22 @@ export default (
     /**
      * The max length for the Labeled Input.
      */
-    _maxlength(): number | null {
+    _maxlength(): number | string {
       if (this.type === 'text' && this.maxlength) {
         return this.maxlength;
       }
 
-      return null;
+      return '';
```

Comment:
Why this change?

Reply:
`InputHTMLAttributes.maxlength` on input has a type of `Numberish | undefined` (`Numberish` is just `number | string`).

I found that an empty string indicates an unlimited `maxlength` in my testing and felt that it was appropriate, but I suppose that we could opt for `undefined` instead of `null` and get the same outcome.